jest.config.js

```diff
@@ -8,4 +8,11 @@ module.exports = {
     // Needed to use @shelf/jest-mongodb preset.
     ...tsJestPreset.transform,
   },
+  collectCoverageFrom: ['./src/**/*.{ts,js}'],
+  coveragePathIgnorePatterns: ['./node_modules/', './tests'],
+  coverageThreshold: {
+    global: {
+      statements: 12, // Increase this percentage as test coverage improves
```

Comment:
I think the current coverage has jumped to above 14%, might as well change it now :P